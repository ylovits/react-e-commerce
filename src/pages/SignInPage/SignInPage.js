import React from "react";
import SignIn from "../../components/SignIn/SignIn";
import SignUp from "../../components/SignUp/SignUp";
import "./SignInPage.scss";

const SignInPage = () => {
  return (
    <div className="container mb-4">
      <div className="row">
        <SignIn />
        <SignUp />
      </div>
    </div>
  );
};

export default SignInPage;
