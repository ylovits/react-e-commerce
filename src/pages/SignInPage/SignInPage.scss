.card-signin,
.card-signup {
  .card-title {
    margin-bottom: 2rem;
    font-weight: 300;
    font-size: 1.5rem;
  }

  .card-body {
    padding: 2rem;
  }

  .form-signin {
    width: 100%;

    .form-label-group {
      position: relative;
      margin-bottom: 1rem;

      input {
        height: auto;
        border-radius: 2rem;
      }

      & > input,
      & > label {
        padding: 0.75rem 1.5rem;
      }

      & > label {
        position: absolute;
        top: 0;
        left: 0;
        display: block;
        width: 100%;
        margin-bottom: 0;
        line-height: 1.5;
        color: #495057;
        border: 1px solid transparent;
        border-radius: 0.25rem;
        transition: all 0.1s ease-in-out;
      }

      input::-webkit-input-placeholder {
        color: transparent;
      }

      input:-ms-input-placeholder {
        color: transparent;
      }

      input::-ms-input-placeholder {
        color: transparent;
      }

      input::-moz-placeholder {
        color: transparent;
      }

      input::placeholder {
        color: transparent;
      }

      input:not(:placeholder-shown) {
        padding-top: calc(0.75rem + 0.75rem * (2 / 3));
        padding-bottom: calc(0.75rem / 3);
      }

      input:not(:placeholder-shown) ~ label {
        padding-top: calc(0.75rem / 3);
        padding-bottom: calc(0.75rem / 3);
        font-size: 12px;
        color: #777;
      }
    }
  }
  .btn {
    font-size: 80%;
    border-radius: 5rem;
    letter-spacing: 0.1rem;
    font-weight: bold;
    padding: 1rem;
    transition: all 0.2s;
  }
  .btn-google {
    color: white;
    background-color: #ea4335;
  }
}

@supports (-ms-ime-align: auto) {
  .form-label-group > label {
    display: none;
  }
  .form-label-group input::-ms-input-placeholder {
    color: #777;
  }
}

@media all and (-ms-high-contrast: none), (-ms-high-contrast: active) {
  .form-label-group > label {
    display: none;
  }
  .form-label-group input:-ms-input-placeholder {
    color: #777;
  }
}
