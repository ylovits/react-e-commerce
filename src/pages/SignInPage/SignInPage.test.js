import React from "react";
import { shallow } from "enzyme";
import SignInPage from "./SignInPage";

it("<SignInPage /> rendered", () => {
  expect(shallow(<SignInPage />)).toMatchSnapshot();
});
