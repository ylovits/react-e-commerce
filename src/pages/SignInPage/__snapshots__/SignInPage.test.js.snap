// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SignInPage /> rendered 1`] = `ShallowWrapper {}`;
